diff --git a/src/cmake/pythonutils.cmake b/src/cmake/pythonutils.cmake
index 5799e29f2..271d354d5 100644
--- a/src/cmake/pythonutils.cmake
+++ b/src/cmake/pythonutils.cmake
@@ -73,6 +73,7 @@ macro (setup_python_module)
     endif ()
 
     pybind11_add_module(${target_name} ${PYLIB_LIB_TYPE} ${lib_SOURCES})
+    message("PYTHON_MODULE_EXTENSION = ${PYTHON_MODULE_EXTENSION}")
 
 #    # Add the library itself
 #    add_library (${target_name} MODULE ${lib_SOURCES})
